src/main/java/migrator/lib/storage/SimplePreferencesStorage.java

```diff
@@ -0,0 +1,35 @@
+package migrator.lib.storage;
+
+import java.util.prefs.BackingStoreException;
+import java.util.prefs.Preferences;
+
+public class SimplePreferencesStorage implements Storage<String> {
```

Comment:
Not in use => remove

Reply:
or move to persistance folder, or remove persistance folder. But Persistance folder might be useful. At least have these classes together